neurolib/models/aln/timeIntegration.py

```diff
@@ -514,7 +514,7 @@ def timeIntegration_njit_elementwise(
             mufi_rhs = (mui - mufi[no]) / tau_inh
 
             # rate has to be kHz
-            IA_rhs = (a * (Vmean_exc - EA) - IA[no, i - 1] + tauA * b * rates_exc[no, i] * 1e-3) / tauA
```

Comment:
Are you sure that this is correct? 

Compare `rates_exc[:, i]` with `Vmean_exc`: 
```python
rates_exc[no, i] = interpolate_values(precalc_r, xid1, yid1, dxid, dyid) * 1e3
Vmean_exc = interpolate_values(precalc_V, xid1, yid1, dxid, dyid)
```

They are the same time step. `rates_exc[no, i-1]` is too old.